conf/test.config

```diff
@@ -18,7 +18,7 @@ params {
     // Limit resources so that this can run on GitHub Actions
     max_cpus   = 2
     max_memory = '6.GB'
-    max_time   = '6.h'
+    max_time   = '8.h'
```

Comment:
Just a reminder to put it back to 2 before merging this PR

Reply:
if we reduce the time, freebayes can possibly not be tested, at least locally it keeps timing out